stellar/deploy-contract.js

```diff
@@ -152,14 +157,14 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
             };
         }
 
-        case 'interchain_token_service': {
-            const gatewayAddress = nativeToScVal(Address.fromString(chain.contracts?.axelar_gateway?.address), { type: 'address' });
-            const gasServiceAddress = nativeToScVal(Address.fromString(chain.contracts?.axelar_gas_service?.address), { type: 'address' });
+        case 'InterchainTokenService': {
+            const gatewayAddress = nativeToScVal(Address.fromString(chain.contracts?.AxelarGateway?.address), { type: 'address' });
+            const gasServiceAddress = nativeToScVal(Address.fromString(chain.contracts?.AxelarGasService?.address), { type: 'address' });
             const itsHubAddress = nativeToScVal(config.axelar?.contracts?.InterchainTokenService?.address, { type: 'string' });
             const chainName = nativeToScVal(chain.axelarId, { type: 'string' });
             const nativeTokenAddress = nativeToScVal(Address.fromString(chain?.tokenAddress), { type: 'address' });
-            const interchainTokenWasmHash = wasmHashToScVal(await uploadContract('interchain_token', options, wallet, chain));
-            const tokenManagerWasmHash = wasmHashToScVal(await uploadContract('token_manager', options, wallet, chain));
+            const interchainTokenWasmHash = wasmHashToScVal(await uploadContract('InterchainToken', options, wallet, chain));
+            const tokenManagerWasmHash = wasmHashToScVal(await uploadContract('TokenManager', options, wallet, chain));
```

Comment:
`wasmHashToScVal` is a weird helper. Make the name more generic like `BytesToScVal` since the implementation is already generic?